core/src/test/scala/org/genivi/sota/core/jsonrpc/HttpTransportSpec.scala

```diff
@@ -36,11 +36,11 @@ class HttpTransportSpec extends JsonRpcSpecBase {
   }
 
   property("fails with error if status code indicates success but result is not a valid json") {
-    forAll(ResultStatusGen, Gen.alphaStr) { (statusCode: StatusCode, entity) =>
+    forAll(ResultStatusGen, Gen.alphaStr.suchThat(s => !s.isEmpty)) { (statusCode: StatusCode, entity) =>
```

Comment:
Without this change, the test fails on entities composed of an empty string. The error is a `NoContentException` instead of a `ParsingFailure`. Is this reasonable? Or should we change the unmarshaller? I couldn't find the unmarshaller used, so do we need to define our own to turn a `NoContentException` into a `ParsingFailure`?